src/tactic/unfold_cases.lean

```diff
@@ -0,0 +1,72 @@
+/-
+Copyright (c) 2020 Dany Fabian. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Dany Fabian
+-/
+
+import tactic.split_ifs
+namespace tactic
```

Comment:
Add module header (after the `import`s) to describe how the module works and what it accomplishes

Reply:
The module doc should go directly after the `import`s and before `namespace` and `open`.